src/tools/wall-tools.js

```diff
@@ -0,0 +1,65 @@
+import {injectMethodCondition, replaceMethod} from '../utils/Injection.js'
```

Comment:
Please rename the file so that it follows the module's naming convention: `WallTools.js`